includes/class-ed-solr.php

```diff
@@ -122,6 +122,22 @@ private function load_dependencies() {
 		 */
 		require_once plugin_dir_path( dirname( __FILE__ ) ) . 'public/class-ed-solr-public.php';
 
+		/**
```

Comment:
Another description comment to be removed from the top of the page

Reply:
done